.github/workflows/publish-release.yml

```
@@ -7,36 +7,88 @@ jobs:
   publish-release:
     name: Publish Release
     runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+      contents: write
+
     steps:
       - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)